rim/templates/rim/home.html

```diff
@@ -46,7 +46,11 @@
                                 {{ e.latest_checkout.timestamp }}
                             {% endif %}
                         </td>
-                        <td><a href="{% url 'edit' pk=e.pk %}">{{ e.serial_no }}</a></td>
+                        <td>
+                            <a href="{% url 'edit' pk=e.pk %}">
+                                {{e.serial_hostname}}
```

Comment:
Just make this match the style used for other Django tags by adding spaces between `e.serial_hostname` and the curly braces. After that's fixed, feel free to merge this branch.